src/components/icon/icon.tsx

```diff
@@ -4,55 +4,30 @@ import iconCache from '../../global/icon-cache/factory';
 import { IconSize } from '../../interface';
 
 /**
- * The recommended icon library for use with Lime Elements is the Windows 10 set
- * from Icons8 (https://icons8.com/icons/windows). This set is included in the
- * relevant Lime products. If you are using Lime Elements in a non-Lime product,
- * you will have to supply your own icons.
- *
- * The size and color of the icon is set in CSS, however there are a few
- * standard sizes defined that can be used with the `size` property.
- *
- * ### Setup
- * To use **@lundalogik/lime-icons8**, the `/assets` folder from
- * __@lundalogik/lime-icons8__ must be made available on the webserver.
- * To use a different icon set, the icons must be placed in a folder structure
- * that looks like this: `assets/icons/<name-of-icon>.svg`
- *
- * If `assets` is placed in the root, no other setup is needed. The icons will
- * be fetched with a relative URL from `/assets/icons/<name-of-icon>.svg`.
+ * Search for an icon and **click on it to copy its name to clipboard**.
+ * <limel-example-icon-finder />
  *
- * If `assets` is placed in a sub-folder somewhere, the easiest way to make the
- * icons available is to use the HTML `base` element:
+ * *******
  *
- * ```
- * <base href="/my/parent/path/">
- * ```
+ * :::important
+ * To install your icon set correctly, please read the [documentations here](/#/).
```

Comment:
```suggestion
 * To install your icon set correctly, please read the [documentation here](/#/).
```

`fixup! docs(README.md): improve documentation about icons`

I don't know where the "s" in "docs" comes from, but there should be no "s" in "documentation", even in plural… 🙂

Reply:
⚡